drivers/hmis/app/graphql/types/hmis_schema/staff_assignment.rb

```diff
@@ -0,0 +1,39 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::StaffAssignment < Types::BaseObject
+    description 'Staff Assignment'
+    field :id, ID, null: false
+    field :user, Application::User, null: false
+    field :household, HmisSchema::Household, null: false
+    field :staff_assignment_type, String, null: false
```

Comment:
I agree with you about "type" implying an enum, but I also remember Gig had concerns about "role" being overloaded too because it sounds like a permissions thing. "staff_assignment_title" (like job title?) is the only other good alternative that I have been able to come up with so far, but I feel hesitant to change things around in the spec that we agreed on before Gig left

Reply:
Feel free to ignore my suggestion about `relationship` above (and here), but that's the other wording that feels vaguely right to me.

Reply:
Thanks, hm I like `relationship` too! I'll go ahead with the change